tsconfig.json

```diff
@@ -13,6 +13,10 @@
     "**/*.d.ts"
   ],
   "compilerOptions": {
-    "lib": ["es2018", "dom", "esnext.asynciterable"]
+    "lib": ["es2018", "dom", "esnext.asynciterable"],
+    "compilorOptions": {
+      "experimentalDecorators": true,
```

Comment:
Is it required? I thought this project inherits from the base tsconfig which has `"experimentalDecorators": true,`.

Reply:
@raymondfeng My vscode doesn't recognize the config in the extended file, it complains experimental decorator feature missing and has red line for all the decorators. So I changed the file. But they should be removed in this PR.